3-networks/modules/dedicated_interconnect/main.tf

```diff
@@ -26,15 +30,15 @@ module "interconnect_attachment1_region1" {
   source  = "terraform-google-modules/cloud-router/google//modules/interconnect_attachment"
   version = "~> 0.2.0"
 
-  name    = "vl-${var.region1_interconnect1_location}-${var.vpc_name}-${var.region1}-cr1"
+  name    = "vl-${var.region1_interconnect1_location}-${var.vpc_name}-${var.region1}-${local.suffix1}"
```

Comment:
I think it would be better to explicitly provide the value, with a variable like `cloud_router_label`

Reply:
good idea